### ISSUE TITLE
website logo shows as question mark

### ISSUE THREAD

**Issue by zhangsj96** [UNLV-SPFG/UNLV-SPFG.github.io#50]
The logo on the top left of the website does not show up. https://unlv-spfg.github.io/assets/images/logo.webp

**Comment by sabaronett** [comment]
@zhangsj96, are you using Safari?

**Comment by sabaronett** [comment]
After checking its [web browser support](https://en.wikipedia.org/wiki/WebP#Web_browsers) (e.g., Safari on iOS 14 and macOS Big Sur), I switched to [WebP](https://en.wikipedia.org/wiki/WebP), which respectively offers better lossy and lossless compression than JPEG and PNG,  for any assets >1 MB to help speed up un-cached page loading over slow and/or mobile networks. But I may switch back to PNG/JPEG if you're experiencing browser rendering issues.

**Comment by zhangsj96** [comment]
Yes, I am on Safari. I found this on **macOS Mojave** using my office desktop.  I switched to chrome then there's no problem. I tried macOS Monterey and iOS 14, and they don't have this issue.

**Comment by sabaronett** [comment]
Thanks for those details, @zhangsj96. Let's leave this issue open for the time being in case anyone else encounters the same issue.
